src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -33,4 +32,8 @@ public CommandResult(String feedbackToUser, List<? extends ReadOnlyPerson> relev
         return Optional.ofNullable(relevantPersons);
     }
 
+    /** The feedback message to be shown to the user. Contains a description of the execution result */
+    public String getFeedbackToUser() {
+        return feedbackToUser;
```

Comment:
If you assign feedbackToUser as `private` class attribute, then you will need to have a separate getter function to get the value of `feedbackToUser`. e.g. 
`string feedback = getFeedBackToUser();`
`return feedback; `

Reply:
Can you phrase the problem in another way? 
I think I did create a public getter, but I did choose not to create a setter as I found no other usage of feedbackToUser other than getting it in some Command functions.
Should I always create a pair of setter and getter when encapsulating data?